### PR TITLE
Fix/cosmetic again

### PR DESCRIPTION
**Pull request by GoOz**
A lot of micro-fix to fit mockups and give our beloved designer some joy.

### CODE REVIEW

File: stylus/ui-app/layouts.styl [hunk 1]
```diff
@@ -40,20 +40,23 @@ $app-2panes
...
 // When you want the sidebar to act like a sticky footer
 $app-2panes-sticky
     @extend $app-2panes
 
     @media (max-width: (1023/basefont)rem)
-        height 'calc(100vh - %s)' % em(102px)
+        height 'calc(100vh - %s)' % em(90px, 16px)
         // Occupy all the available height minus the heights of the Cozy-bar
-        // (52px) on top & the navigation (50px) on bottom
+        // (40px) on top & the navigation (50px) on bottom
```
Comment (m4dz) [inline stylus/ui-app/layouts.styl]: I ask myself about the flexibility of this one… do we really can rely on a fixed value for cozy-bar and bottom avout their height? Can't it be broken unexpectingly?

Comment (GoOz) [inline stylus/ui-app/layouts.styl]: I don't like it either. Between that and the z-index, I'm struggling to find something less hazardous. So far i can't find any good idea but I'm open to hear new ones. 
Anyway, until then we can use it as it is, but we clearly should find a better way.

**Comment by y-lohse** [comment]
I know these are lots of micro-fixes, but shouldn't there be a mention of some in the changelog?

**Comment by enguerran** [comment]
At least something like "a lot of minor UI fixes"

**Comment by GoOz** [comment]
Damn right. Just do what i say, not what i do xD

**Pull request merged**